db/migrate/20240118134238_create_users.rb

```diff
@@ -2,7 +2,7 @@ class CreateUsers < ActiveRecord::Migration[7.1]
   def change
     create_table :users do |t|
       t.string :name
-      t.string :password
+      
       t.timestamps
```

Comment:
If a user is to sign in... how can they do so without a password? In our [frontend](https://github.com/leloufadel/front-end-capstone/pull/7) we still call and use passwords to make our calls and fill our user details. So every time we create a user, we still give it a "password" even though the user in our database doesn't actually have a password anymore thanks to this change.

Why remove it?
![image](https://github.com/leloufadel/full-stuck-project/assets/91759364/dfc52dc7-da66-460d-81bc-0d56a71f4cd1)

Reply:
Good question. I'd didn't notice. @AlselwiAisha can explain it to us tomorrow. I think she's not available right now